code/src/chunk.rs

```diff
@@ -79,14 +73,34 @@ impl fmt::Display for Chunk {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use crate::chunk_type::ChunkType;
     use std::str::FromStr;
 
     fn testing_chunk() -> Chunk {
-        let chunk_type = ChunkType::from_str("RuSt").unwrap();
-        let data: Vec<u8> = "This is where your secret message will be!"
-            .bytes()
+        let data_length: u32 = 42;
+        let chunk_type = "RuSt".as_bytes();
+        let message_bytes = "This is where your secret message will be!".as_bytes();
+        let crc: u32 = 2882656334;
+
+        let chunk_data: Vec<u8> = data_length
+            .to_be_bytes()
+            .iter()
+            .chain(chunk_type.iter())
+            .chain(message_bytes.iter())
+            .chain(crc.to_be_bytes().iter())
+            .copied()
```

Comment:
This is giving away an implementation strategy that I'd like the reader to discover themselves.